spaceWeatherApi/AppSettings.cs

```diff
@@ -0,0 +1,34 @@
+﻿using System;
+using System.Collections.Generic;
+using System.Linq;
+using System.Text;
+using System.Threading.Tasks;
+
+namespace spaceWeatherApi
```

Comment:
You can use pascal case for the namespace which follows C# standards
```suggestion
namespace SpaceWeatherApi
```